com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/Fontc.java

```diff
@@ -160,15 +161,24 @@ public class Glyph {
     public static long FontDescToHash(FontDesc fontDesc) {
         FontDesc.Builder fontDescbuilder = FontDesc.newBuilder();
 
-        fontDescbuilder.mergeFrom(fontDesc)
-                       .setMaterial("")
-                       .clearOutlineAlpha()
-                       .clearShadowAlpha()
-                       .clearShadowX()
-                       .clearShadowY()
-                       .clearAlpha();
-
-        return fontDescbuilder.build().hashCode();
+        fontDescbuilder.mergeFrom(fontDesc);
+        FontDesc desc = fontDescbuilder.build();
+
+        // the list of parameters which affect the glyph_bank
+        String result = ""
+            + desc.getFont()
+            + desc.getSize()
+            + desc.getAntialias()
+            + desc.getOutlineWidth()
+            + desc.getShadowBlur()
+            + desc.getExtraCharacters()
+            + desc.getOutputFormat()
+            + desc.getAllChars()
```

Comment:
Will this work with e.g. a large chinese font? I mean, will there be a UTF problem or so?

Reply:
yes, it's fine